### DISCUSSION TITLE
Title in header of each block

### DISCUSSION THREAD

**Discussion by ricoroodenburg** [postmanlabs/postman-flows#347] [Ideas]
Hi,

It should be very nice if it is possible to give every block a title.
Adds this title on top of a block, above the action type.

![Picture2](https://user-images.githubusercontent.com/79546116/178107516-4b9c2bc5-3471-4b87-92d6-5a8ad89df19a.png)

You can already add a description / summary, but this is not direct visible.

**Comment by mattnield** [comment]
I'd second this, it would make larger flows _much_ more readable!
